src/jit/emitarm64.cpp

```diff
@@ -6220,6 +6335,102 @@ void emitter::emitIns_S_R(instruction ins, emitAttr attr, regNumber reg1, int va
 
 /*****************************************************************************
  *
+ *  Add an instruction referencing consecutive stack-based local variable slots and two registers
+ */
+void emitter::emitIns_S_S_R_R(
+    instruction ins, emitAttr attr1, emitAttr attr2, regNumber reg1, regNumber reg2, int varx, int offs)
+{
```

Comment:
@briansull Shouldn't emitIns_S_R & emitIns_R_S be emitIns_S_R_I & emitIns_R_S_I repsectively by the same argument?  Do you want me to change the names of the new ones and/or the old ones?